src/LIBMONGOC_VERSION_CURRENT

```diff
@@ -1 +1 @@
-1.18.0-20210426+git724f1672f2
+1.18.1-20210621+git4069b7984d
```

Comment:
Not sure if it's worth it, but I've updated this manually to `1.18.0-...`.

Reply:
No longer relevant as I removed the submodule bump after rebasing.